### ISSUE TITLE
statfs hard-coded

### ISSUE THREAD

**Issue by donporter** [oscarlab/graphene#786]
Our implementation of statfs returns hard-coded data rather than something useful.

**Comment by dimakuv** [comment]
This is true: https://github.com/oscarlab/graphene/blob/af2f7809b36945382a535f95c2af7a2b32a2059a/LibOS/shim/src/sys/shim_stat.c#L158-L163

However, the priority of this is very low. Does any application truly care?
